### DISCUSSION TITLE
How to detect page refresh in Quasar?

### DISCUSSION THREAD

**Discussion by donPuerto** [quasarframework/quasar#10475] [General - Components / Directives / etc] [answered]
Is there a way in Quasar on how to detect page refresh, in the browser?

**Comment by pikil** [comment, marked as answer]
@donPuerto, if you mean before someone hits 'Refresh' button, then you can try window.onbeforeunload which is triggered right after the window object gets unloaded:
https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload
